rails-app/universal/app/views/layouts/_navigation.html.erb

```diff
@@ -9,7 +9,7 @@
                 <i class="fa fa-home"></i> <span class="nav-label">Homepage</span>
                 <% end %>
             </li>
-            <li class="<%= 'active' if user_signed_in? %>">

```

Comment:
I've made some changes here can you please rebase pr and resend. There is a bug when current_user is not signed_in?